### ISSUE TITLE
Allow the user to provide a custom SCEP CA cert

### ISSUE THREAD

**Issue by samcon** [micromdm/micromdm#232]
Right now the SCEP CA Cert is generated by the server once. 
It would be beneficial to supply the SCEP CA Cert via command line.

**Comment by jessepeterson** [comment]
I agree with the idea behind this request. E.g. if you wanted the SCEP CA to be a part of your Org's CA chain in some interesting way (i.e. actually an intermediate).
